spec/mailers/previews/shipment_preview.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+require 'open_food_network/scope_variant_to_hub'
+
+module Spree
+  class ShipmentPreview < ActionMailer::Preview
+    def shipped
+      ShipmentMailer.shipped_email(Shipment.last)
```

Comment:
Shall we provide a sample shipment here instead of querying the database? Okay with me to leave it for now and only change if this becomes a problem.

I think that we have shipments in the sample data and therefore this should work (after loading the sample data).

Reply:
My idea here, when using `.last` is that the tester could easily test the email by creating an order with shipment via the interface (and not by modifying the code). That's what I've done, and I found this efficient.

Reply:
Okay, that's a nice idea. :+1: